components/SpacesCard.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Paper, Container, Box, Typography } from "@material-ui/core";
+import { useTheme } from "@material-ui/core/styles";
+import VideocamOffIcon from "@material-ui/icons/VideocamOff";
+import MicIcon from "@material-ui/icons/Mic";
+import ChatIcon from "@material-ui/icons/Chat";
+import PeopleIcon from "@material-ui/icons/People";
+import LibraryMusicIcon from "@material-ui/icons/LibraryMusic";
+
+export default function SpacesCard(props) {
+  const theme = useTheme();
+
+  return (
+    <Paper className={`flex flex-col`}>
+      <Box className="flex flex-grow justify-between ">
+        <Box color={theme.palette.primary.dark} className="text-left p-3">
+          <h2 className="text-xl">{props.spaceName}</h2>
+          <p className="text-sm text-gray-500">{props.description}</p>
```

Comment:
I'd recommend using typography instead since tailwind is gonna reset styles for just h2 tags. Also destructure props again; not just here, everywhere.